test-utils/src/lib.rs

```diff
@@ -254,13 +250,14 @@ impl Test {
 // HELPER FUNCTIONS
 // ================================================================================================
 
-/// Takes an array of u64 values and builds a stack, preserving their order and converting them to
-/// field elements.
-pub fn convert_to_stack(values: &[u64]) -> [Felt; STACK_TOP_SIZE] {
-    let mut result = [Felt::ZERO; STACK_TOP_SIZE];
-    for (&value, result) in values.iter().zip(result.iter_mut()) {
-        *result = Felt::new(value);
-    }
+/// Converts an array of Felts into u64
+pub fn stack_to_int(values: &[Felt]) -> Vec<u64> {
+    values.iter().map(|e| (*e).as_int()).collect()
+}
+
+pub fn stack_to_size(values: &[u64]) -> Vec<u64> {
```

Comment:
minor nit: I'd call this `stack_top_to_ints()`.

Reply:
renamed

Reply:
I think there was a small typo here: was `stack_to_top_ints` instead of `stack_top_to_ints` - but I think that's fine.